sapi/fpm/tests/log-suppress-output-request-body.phpt

```diff
@@ -0,0 +1,113 @@
+--TEST--
+FPM: Test URIs are not excluded from access log when there is a request body
+--SKIPIF--
+<?php include "skipif.inc"; ?>
+--FILE--
+<?php
+
+require_once "tester.inc";
+
+$testScript = <<<EOT
+<?php
+echo strlen(file_get_contents('php://input'));
+EOT;
+$s = __DIR__ . '/health_check.php';
+file_put_contents($s, $testScript);
```

Comment:
This should be done in the tester.

Reply:
Why don't just pass the testScript as a second parameter to the tester constructor?

Reply:
Can I make `getFile` which calculates the filename public then? I need to know it to create the config

Reply:
I've just settled on hardcoding "log-suppress-output-request-body.src.php", not the worst thing in the world